global_install_scripts/install.bash

```diff
@@ -199,6 +200,60 @@ function run() {
 	fi
     }
 
+    function add_talisman_home_as() {
+    # set TALISMAN_HOME path for user if user opts for it
+    #   user has option to set TALISMAN_HOME in .bashrc or .profile
+    #   user can opt out of auto-setup of TALISMAN_HOME to set it up later manually
+    TALISMAN_SETUP_DIR="$1"
+    echo "Setting up TALISMAN_HOME in path"
+
+    if [ -n "${TALISMAN_HOME:-}" ]; then
+        echo -e "TALISMAN_HOME is already set\n"
+        return 0;
+    fi
+
+    BASHRC_OPT="Set TALISMAN_HOME in ~/.bashrc"
+    BASHPROFILE_OPT="Set TALISMAN_HOME in ~/.bash_profile"
+    PROFILE_OPT="Set TALISMAN_HOME in ~/.profile"
+    SELFSETUP_OPT="I will set it later"
+    TALISMAN_HOME=""
+
+    echo -e "\n\nPLEASE CHOOSE WHERE YOU WISH TO SET TALISMAN_HOME VARIABLE (Enter option number): "
+    options=(${BASHRC_OPT} ${BASHPROFILE_OPT} ${PROFILE_OPT} ${SELFSETUP_OPT})
```

Comment:
Are the options being displayed to the user at any point ?
Is the user's selection of option being read ?